jaymzh/pius
# [pius-report] Support for storing that someone else won't sign us

libpius/state
* Add new version 3 of state file, including new metadata to identify it
* v3 state file includes both 'inbound' and 'outbound' info now
* Add better support for detecting the various versions of the statefile and
converting them to v2
* Added unittests
* add debug
* drop all classmethods and ensure people can and do instantiate us

pius
* instantiate SignState(libpius/state)

pius-report
* Add new feature to re-ask about all keys on a keyring
* Add new feature to update state for a specific key
* Add support for new libpius/state features (inbound signing info)
* Tell Signer to sort keys for us

test/libpius/test_state.py
* unittests for statefile conversion
